pages/account/index.vue

```diff
@@ -26,7 +26,7 @@
                 <b-row class="flex-md-row">
                   <b-col cols="12">
                     <div
-                      v-if="likedItems && likedItems.length === 0"
+                      v-if="$fetchState.pending"
```

Comment:
This is going to re-introduce the wait on the private & public set calls being fully loaded before showing the likes tab.

Instead maybe check that there is a likesId available. _or_ whenever a user visits their account page ensure a likes set is created if there isn't one already.